CMakeLists.txt

```diff
@@ -13,3 +14,35 @@ set (CMAKE_RUNTIME_OUTPUT_DIRECTORY ..)
 add_executable(TransferBench src/TransferBench.cpp)
 target_include_directories(TransferBench PRIVATE src/include)
 
+find_package(ROCM 0.8 REQUIRED PATHS ${ROCM_PATH})
+include(ROCMInstallTargets)
+include(ROCMCreatePackage)
+set(ROCMCHECKS_WARN_TOOLCHAIN_VAR OFF)
+
+set(PACKAGE_NAME TB)
+set(LIBRARY_NAME TransferBench)
+
+rocm_package_setup_component(${PACKAGE_NAME}
+    LIBRARY_NAME ${LIBRARY_NAME}
+    PACKAGE_NAME ${PACKAGE_NAME}
+)
```

Comment:
I don't think that this is necessary, the default packages should be sufficient.